Sources/Rendering/Core/ImageCPRMapper/index.d.ts

```diff
@@ -157,6 +158,59 @@ export interface vtkImageCPRMapper extends vtkAbstractMapper3D {
 	 */
 	setDirectionMatrix(mat: mat3): boolean;
 
+	/**
+	 * Enable or disable Maximum Intensity Projection
+	 * Can be configured to use other modes of projection
+	 * @see getMipSlabThickness
+	 * @see getMipNumberOfSamples
+	 * @see getMipMode
+	 */
+	getUseMip(): boolean;
+
+	/**
+	 * @see getUseMip
+	 * @param useMip
+	 */
+	setUseMip(useMip: boolean): boolean;
+
+	/**
+	 * Thickness of the MIP slab in image coordinates (NOT in voxels)
+	 * Usually in millimeters if the spacing of the input image is set from a DICOM
+	 */
+	getMipSlabThickness(): number;
+
+	/**
+	 * @see getMipSlabThickness
+	 * @param mipSlabThickness
+	 */
+	setMipSlabThickness(mipSlabThickness: number): boolean;
+
+	/**
+	 * Total number of samples of the volume done by the MIP mode
+	 * Using an odd number is advised
+	 * If this number is even, the center of the slab will not be sampled
+	 */
+	getMipNumberOfSamples(): number;
+
+	/**
+	 * @see getMipNumberOfSamples
+	 * @param mipNumberOfSamples
+	 */
+	setMipNumberOfSamples(mipNumberOfSamples: number): boolean;
+
+	/**
+	 * The different modes of projection
+	 * Even if the name of the mode is MIP (which implies a maximum intensity projection),
+	 * other modes are available such as MinIP and Average Intensity Projection
+	 */
+	getMipMode(): MipMode;
+
+	/**
+	 * @see getMipMode
+	 * @param mipMode
+	 */
+	setMipMode(mipMode: MipMode): boolean;
```

Comment:
Conventions say that acronyms must not be camel-cased. it is MIP or mip, but never Mip